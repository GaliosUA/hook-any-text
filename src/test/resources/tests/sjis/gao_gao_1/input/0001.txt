89B482CC96BC914F82CD040020040181420D90BC905688E491E58A7782CC8251944E90B68142