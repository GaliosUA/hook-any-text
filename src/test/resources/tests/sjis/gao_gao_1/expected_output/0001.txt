["俺の名前は柴田　祐二。
西新井大学の２年生。"]